Spawn-App-iOS-SwiftUI/Views/ProfileView.swift

```diff
@@ -13,55 +13,147 @@ struct ProfileView: View {
+
+                    // Logout Button
+                    Button(action: {
+                    }) {
+                        Text("Log Out")
                             .font(.headline)
-                        ForEach(friendTags) { tag in
-                            HStack {
-                                Circle()
-                                    .fill(tag.color)
-                                    .frame(width: 10, height: 10)
-                                Text(tag.displayName)
-                            }
-                        }
+                            .foregroundColor(.white)
+                            .padding()
+                            .frame(maxWidth: 170)
+                            .background(profilPicPlusButtonColor)
+                            .cornerRadius(20)
                     }
+                    .padding(.horizontal)
+                    
+                    
+                    // commented out friend tags and last location for now as it's not included in the Figma Design
+                    
+                    // Friend Tags
+//                    if let friendTags = appUser.friendTags, !friendTags.isEmpty {
+//                        Text("Friend Tags:")
+//                            .font(.headline)
+//                        ForEach(friendTags) { tag in
+//                            HStack {
+//                                Circle()
+//                                    .fill(tag.color)
+//                                    .frame(width: 10, height: 10)
+//                                Text(tag.displayName)
+//                            }
+//                        }
+//                    }
                     
                     // Last Location
-                    if let lastLocation = appUser.lastLocation {
-                        Text("Last Location: \(lastLocation.locationName)")
-                            .font(.subheadline)
-                    }
+//                    if let lastLocation = appUser.lastLocation {
+//                        Text("Last Location: \(lastLocation.locationName)")
+//                            .font(.subheadline)
+//                    }
                 }
                 .padding()
-                .navigationTitle("\(appUser.firstName ?? appUser.username)'s Profile")
+//                .navigationTitle("\(appUser.firstName ?? appUser.username)'s Profile")
             }
+            .background(universalBackgroundColor)
+        }
+    }
+}
+
+
+struct ProfileField: View {
+    let label: String
+    let value: String
+
+    var body: some View {
+        HStack {
+            Text(label)
+                .font(.headline)
+                .frame(width: 100, alignment: .leading)
+            Spacer()
+            Text(value)
+                .font(.body)
+                .multilineTextAlignment(.trailing)
+        }
+    }
+}
+
+struct BioField: View {
+    let label: String
+    @Binding var bio: String
```

Comment:
Great job, but would prefer if these were in a new directory `Views/SubComponents/ProfileFieldViews`, to stick with our current codebase conventions of one struct per file.